cli/src/plugins/javadoc/index.ts

```diff
@@ -323,6 +339,32 @@ function makeClassDocPageBody(args: MakeSectionArgs): Node[] {
           .flat(1),
     }),
 
+    ...makeSection({
+      ...args,
+      depth,
+      title: "Enum Constant Detail",
+      shouldMakeSection: () => doc.enumConstants.length !== 0,
+      makeBody: () =>
+        doc.enumConstants
+          .map((doc) => [
+            project.declareEntity({
+              canonicalName: doc.containingClass?.asString + "_" + doc.name,
```

Comment:
There should be a fully qualified name for the enum (something like `com.example.somePackage.SomeClass.ENUM_VALUE`). If not, I would create a helper function to `getCanonicalNameFromEnumConstantsDoc` (or something) so that the two places that need it are sure to get it right.

Reply:
I believe that's just `qualifiedName` in this case, but as I mentioned above, I'm curious if there's a "correct" way to know the canonical name, or if we're just looking for the field that matches how the Javadoc displays.